src/server.ts

```diff
@@ -4,19 +4,20 @@ import { getConfig } from './config';
 const port = getConfig().server.port;
 const address = getConfig().server.listenAddress;
 const debug = getConfig().server.debug;
+const logger = {
+  transport:
+    process.env.NODE_ENV === 'development' || debug
+      ? {
+          target: 'pino-pretty',
+          options: {
+            translateTime: 'HH:MM:ss Z',
+          },
+        }
+      : undefined,
+};
 
 const server = app({
-  logger: {
-    transport:
-      process.env.NODE_ENV === 'development' || debug
-        ? {
-            target: 'pino-pretty',
-            options: {
-              translateTime: 'HH:MM:ss Z',
-            },
-          }
-        : undefined,
-  },
+  logger: process.env.NODE_ENV === 'production' ? false : logger,
```

Comment:
doesn't it ALWAYS disable logging  in production env even with debug set to true?